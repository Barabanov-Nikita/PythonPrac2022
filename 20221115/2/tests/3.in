from decimal import Decimal
class D:
  num = Num()
print(D().num)
d = D()
print(d.num)
d.num = Decimal("10.99999999999999999999999999")
print(d.num)
d.num = 7.123
print(d.num)