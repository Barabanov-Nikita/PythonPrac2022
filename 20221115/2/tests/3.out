0
0
10.99999999999999999999999999
7.123
